site/content/overview/product-telemetry.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Product Telemetry"
+description: "Learn how NGINX Gateway Fabric collects product telemetry to identify areas to focus development on improving."
+weight: 500
+toc: true
+---
+
+## Overview
+
+NGINX Gateway Fabric collects telemetry by default, which allows its developers to understand what features are most popular with its user base. This data is used to triage development work, prioritizing features and functionality that will positively impact the most people.
+
+Telemetry data is collected once every 24 hours and sent to a service managed by F5. Personally identifiable information (PII) is **not** collected. The list of data points that are collected can be seen below.
+
+**If you would prefer to not have data collected, you can [opt-out](#opt-out) when installing NGINX Gateway Fabric.**
+
+## Collected data
+
+- **Kubernetes Platform and Version:** the Kubernetes platform and version that NGINX Gateway Fabric is running on.
+- **Platform Architecture:** the architecture that the Kubernetes environment is running on.
+- **Cluster UID:** the UID of the `kube-system` Namespace in the cluster that NGINX Gateway Fabric is running in.
+- **Cluster Node Count:** the number of Nodes in the cluster.
+- **Deployment UID:** the UID of the NGINX Gateway Fabric Deployment.
+- **Deployment Replica Count:** the count of NGINX Gateway Fabric Pods.
+- **Image Build Source:** whether the image was built by GitHub or locally (values are `gha`, `local`, or `unknown`). The source repository of the images is **not** collected.
+- **NGINX Modules:** a list of installed NGINX modules.
+- **Deployment Flags:** a list of NGINX Gateway Fabric Deployment flags that are specified by a user. The actual values of non-boolean flags are **not** collected; we only record that they are either `default` or `user-defined`.
+- **Count of Gateway Resources:** the total count of resources related to NGINX Gateway Fabric. This includes `GatewayClasses`, `Gateways`, `HTTPRoutes`, `Secrets`, `Services`, and `Endpoints`. The data within these resources is **not** collected.
```

Comment:
this we collect both built-in k8s resources and Gateway API resources (and soon our own resources), better to remove Gateway
```suggestion
- **Count of Resources:** the total count of resources relevant for NGINX Gateway Fabric. This includes `GatewayClasses`, `Gateways`, `HTTPRoutes`, `Secrets`, `Services`, and `Endpoints`. The data within these resources is **not** collected.
```